notebooks/finetuning/convfinqa_finetuning_wandb.ipynb

```diff
@@ -358,21 +358,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Once your model completes training, you can call it via co.chat() and pass your custom model id. Please note, the model id is the id returned by the fine-tuned model object + `-ft` suffix. `co.chat()` uses no preamble by default for fine-tuned models. You can specify a preamble using the preamble parameter, if you like. In this case, we wont specify any preamble and follow the convention set in the training data."
+    "Once your model completes training, you can call it via co.chat() and pass your custom model id. Please note, the model id is the id returned by the fine-tuned model object + `-ft` suffix. `co.chat()` uses no preamble by default for fine-tuned models. You can specify a preamble using the preamble parameter, if you like. In this case, we wont specify any preamble and follow the convention set in the training data. The response object returned by the API is described in detail [here](https://docs.cohere.com/reference/chat#response). "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 43,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "#### Model response ####\n",
-      "add(189, 68)\n",
-      "########################\n"
+      "#### Finetuned model response ####\n",
```

Comment:
I think the #s here are breaking it, seems like they're treated as markdowns, I think you need to change these

Reply:
so the top ## end up making this a small heading which is good i think. this bottom ones were breaking it. with --- they should render as a separation line
<img width="992" alt="image" src="https://github.com/user-attachments/assets/d0b226cf-476f-42dc-a997-05c5e11953f2">

Reply:
you need to show exactly as what you  print  out in the code, otherwise it's confusing.

Reply:
updated, let me know what you think

Reply:
[this](https://cohere-preview-ba9f43e2-2712-4cf4-95df-ad9f21448a25.docs.buildwithfern.com/page/convfinqa-finetuning-wandb) is the manually fixed preview.

if this looks good, we would not need this PR.